src/Makevars

```diff
@@ -5,9 +5,9 @@
 # The full license is in the file LICENSE, distributed with this software.
 
 # Default make file
-PKG_CPPFLAGS=-std=c++17
+PKG_CPPFLAGS=-std=c++17 -I$(R_HOME)/../../Library/include
```

Comment:
I can't think of a better way to address the Library prefix of conda, than by `$(R_HOME)/../../`.

Reply:
We're having this issue on Windows only right?
Or can you reproduce in a different OS?